File: 69-sqrtx/sqrtx.py
class Solution:
    def mySqrt(self, x: int) -> int:
        if x<2:
            return x

        a=1
        while((a*a)<x):
            a+=1
        
        if((a*a)==x):
            return a
        else:
            return a-1
        

File: 69-sqrtx/sqrtx.js
/**
 * @param {number} x
 * @return {number}
 */
var mySqrt = function(x) {
    let i=0;
    
    while(true){
        if(i*i===x) return i;
        else if(i*i>x) return i-1;
        i++;
    }
};